Improve wifi coex doc (#952)
nrfconnect/sdk-nrfxlib: nrf_802154/doc/wifi_coex_module.rst

The request can contain multiple pieces of information, such as the operation the protocol intends to perform or the priority level of that operation.
The PTA then grants or denies access to the RF medium for that request.

> The nRF 802.15.4 Radio Driver uses abstract :ref:`mpsl_cx` provided by :ref:`mpsl` with following limitations:

Reviewer: ```suggestion
The nRF 802.15.4 Radio Driver uses the abstract :ref:`mpsl_cx` provided by :ref:`mpsl` with the following limitations:
```

> * It does not use radio operation priorities, as the each operation is requested with a priority value of ``1``.

Reviewer: ```suggestion
* It does not use radio operation priorities, as each operation is requested with a priority value of ``1``.
```

> * When a PTA denies an operations, only TX is aborted, denying RX or idle listening has no effect.

Reviewer: ```suggestion
* When a PTA denies an operations, only the TX operation is aborted.
  The denial of an RX operation or of *idle listening* has no effect.
```
